VTEX - Catalog API.json

```diff
@@ -17036,7 +17036,7 @@
                 }
             }
         },
-        "/api/catalog/pvt/specification/nonstructured/{skuId}": {
+        "/api/catalog/pvt/specification/nonstructured/skuId={skuId}": {
```

Comment:
Se for uma query, ela não fica no path, é só adicionar nos parms como query e required

Reply:
Esse `skuId=` já indica que é uma query, se fosse um path param seria `{skuId}`

Reply:
Te chamei no Slack

Reply:
Ajustado! @karenkrieger